lambda/custom-erc721/scripts/.env

```diff
@@ -8,4 +8,7 @@ LAMBDA_ADDRESS =
 // The email of the user who will be whitelisted and minted a token.
 // You can also use an eth address here 
 // after modifying the whitelistAndMint.js script.
-USER_EMAIL = 
\ No newline at end of file
+USER_EMAIL = 
+
+
+API_HOST =
```

Comment:
We don't need to add here and other places.

Reply:
fixed